src/Images.jsx

```diff
@@ -225,6 +225,16 @@ class Images extends React.Component {
                     return false;
                 });
         }
+
+        filtered.sort((a, b) => {
+            // User images are in front of system ones
+            if (this.props.images[a].isSystem !== this.props.images[b].isSystem)
+                return this.props.images[a].isSystem ? 1 : -1;
+            const name_a = this.props.images[a].repoTags ? this.props.images[a].repoTags[0] : "";
+            const name_b = this.props.images[b].repoTags ? this.props.images[b].repoTags[0] : "";
+            return name_a > name_b ? 1 : -1;
```

Comment:
Nitpick: This will sort images without tags before images with a tag, which feels kind of backwards -- the tagged ones are the important ones, the untagged ones are usually just some intermediate layer noise.

Reply:
Great point! I've fixed this. Also I plan to hide these images by default, but need to talk to @andreasn about some usable design.